CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+> [!NOTE]
```

Comment:
Do you envision this to be a temporary place to hold this note? Will we constantly be updating this when gen-ai instrumentations are added? Perhaps we put all gen-ai instrumentations in the future in a specific folder and outline that components in that folder will also have individual changelogs?

Reply:
I think that in the long term we should support individual release for all components (including existing non-gen-ai ones). 

I.e. the central CHANGELOG would disappear. 

I'm going to steal the following approach from [js-contrib](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/ee5c584c943eb8a3de0566843a42784f16c0f31f/CHANGELOG.md?plain=1#L4):

> ~~As of v0.25.1 (2022-01-24) changelog content has moved to separate CHANGELOG.md files for each package.~~ Use [this search for a list of all CHANGELOG.md files in this repo](https://github.com/search?q=repo%3Aopen-telemetry%2Fopentelemetry-js-contrib+path%3A**%2FCHANGELOG.md&type=code).